.github/CODEOWNERS

```
@@ -11,4 +11,4 @@
 # .pre-commit-config.yaml @wusatosi # Add other project owners here
 # .markdownlint.yaml @wusatosi # Add other project owners here
 
-*  @bretbrownjr @dietmarkuehl @neatudarius @steve-downey @wusatosi @bemanproject/core-reviewers
+*  @bretbrownjr @dietmarkuehl @steve-downey @wusatosi
```

```suggestion
*  @bretbrownjr @dietmarkuehl @neatudarius @steve-downey @wusatosi
```

I didn't want to remove myself, sorry